tachyon/crypto/hashes/sponge/poseidon2/poseidon2_internal_matrix_unittest.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022 The Plonky3 Authors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.plonky3 and the LICENCE-APACHE.plonky3
+// file.
+
+#include "tachyon/crypto/hashes/sponge/poseidon2/poseidon2_internal_matrix.h"
+
+#include "gtest/gtest.h"
+
+#include "tachyon/math/finite_fields/mersenne31/mersenne31.h"
+#include "tachyon/math/finite_fields/test/finite_field_test.h"
+#include "tachyon/math/matrix/prime_field_num_traits.h"
+
+namespace tachyon::crypto {
+
+using F = math::Mersenne31;
```

Comment:
You mean anonymous namespace? If so, AFAIK, since anonymous namespace affects only when you creating a symbol. This is just an alias I think it has nothing to do with where you use `using X`!